Mathlib/Analysis/Calculus/ParametricIntegral.lean

```diff
@@ -63,31 +63,32 @@ variable {α : Type*} [MeasurableSpace α] {μ : Measure α} {𝕜 : Type*} [IsR
   [NormedAddCommGroup E] [NormedSpace ℝ E] [NormedSpace 𝕜 E] [CompleteSpace E] {H : Type*}
   [NormedAddCommGroup H] [NormedSpace 𝕜 H]
 
+variable {bound : α → ℝ} {ε : ℝ} (ε_pos : 0 < ε)
```

Comment:
Please don't move explicit arguments, especially assumptions, to `variable`s: this way `theorem` statements below miss important parts.